runtime/src/blueoil_image.cpp

```diff
@@ -133,26 +146,39 @@ Tensor ResizeVertical_BiLinear(const Tensor &tensor, const int height) {
   const int channels  = shape[2];
   const int width = srcWidth;
   Tensor dstTensor({height, width, channels});
-  float yScale = static_cast<float> (height) / static_cast<float>(srcHeight);
-  int ySrcWindow = std::floor(1/yScale);
-  ySrcWindow = (ySrcWindow < 2)? 2 :ySrcWindow;
-  for (int dstY = 0 ; dstY < height ; dstY++) {
-    for (int dstX = 0 ; dstX < width ; dstX++) {
+  const float yScale = static_cast<float> (height) / static_cast<float>(srcHeight);
+  const int ySrcWindow = (yScale < 1.0)?(1.0f / yScale):1.0;
```

Comment:
Could you insert a space before/after ? and :?

Reply:
https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947#diff-37230609db7212ca72dd52076761c190L151